gulpfile.js

```diff
@@ -0,0 +1,306 @@
+/******************************************************
+ * PATTERN LAB NODE
+ * EDITION-NODE-GULP
+ * The gulp wrapper around patternlab-node core, providing tasks to interact with the core library and move supporting frontend assets.
+******************************************************/
+const gulp = require('gulp');
+const path = require('path');
+const browserSync = require('browser-sync').create();
+const sass = require('gulp-sass');
+const eyeglass = require('eyeglass');
+const argv = require('minimist')(process.argv.slice(2));
+const chalk = require('chalk');
+const concat = require('gulp-concat');
```

Comment:
Are we not?
Sure, the location and name of `pattern-scaffolding.css` may change, but I'd have thought we will have one.